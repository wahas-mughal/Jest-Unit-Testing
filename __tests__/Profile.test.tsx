import { render } from '@testing-library/react-native';
import React from 'react';
import 'react-native';
import ProfileScreen from '../src/Profile';
import { expect, test } from '@jest/globals';

test('Profile Screen Snapshot', () => {
  const snap = render(<ProfileScreen />).toJSON();
  expect(snap).toMatchSnapshot();
});
