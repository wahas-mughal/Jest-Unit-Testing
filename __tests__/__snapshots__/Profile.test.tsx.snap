// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Screen Snapshot 1`] = `
<View>
  <Text>
    Profile
  </Text>
</View>
`;
